src/Model/Config/Source/AdminUsernameList.php

```diff
@@ -5,11 +5,8 @@
  */
 namespace Vaimo\AdminAutoLogin\Model\Config\Source;
 
-use function __;
-
-class AdminUser implements \Magento\Framework\Option\ArrayInterface
+class AdminUsernameList
```

Comment:
The ArrayInterface is actually needed, for the dropdown in module's admin config when selecting the admin user to automatically log in as.

So please refactor like this:
* Move this model to Vaimo/AdminAutoLogin/Model namespace, it's good as this
* Recreate Model/Config/Source/AdminUser, so that its toOptionArray implementation uses AdminUsernameList

Reply:
Wow didn't know that 😁. That's a smart idea, thank you! Regarding the indentation I'm not sure I agree.. But maybe we should change the whole styling of this module anyways. It's kind of weird / out dated.

Reply:
We should style according to Magento coding standard guidelines with phpcs!